igvm_defs/src/lib.rs

```diff
@@ -1046,7 +1047,7 @@ pub enum VbsDigestAlgorithm {
 
 /// The signature algorithm to use for VBS measurement.
 #[open_enum]
-#[derive(AsBytes, FromBytes, Debug, Clone, Copy, PartialEq, Eq, Hash)]
+#[derive(AsBytes, FromBytes, FromZeroes, Debug, Clone, Copy, PartialEq, Eq, Hash)]
```

Comment:
Ditto above^
I don't understand how "FromZeroes" makes sense here - the enum only has one value, and it isn't zero?

Reply:
Same as above - this bound is fine, but lets define 0 as INVALID.